### PR TITLE
[CAS-380] Small readme update

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
[CAS-380](https://stream-io.atlassian.net/browse/CAS-380)

### Description

Snall update in the documentation for the README. 

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- Changelog updated with client-facing changes. **no need**
- New code is covered by unit tests. **no need**
- Comparison screenshots added for visual changes. **no need**
- [x] Reviewers added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,10 @@ This repo includes a fully functional example app featuring threads, reactions,
 git clone git@github.com:GetStream/stream-chat-android.git
 ```
 
-Next, download Android Studio and open up the `stream-chat-android` folder. You'll want to run the `stream-chat-android-sample` app.
+Next, download Android Studio and open up the `stream-chat-android-ui-components-sample` folder. You'll want to run the `stream-chat-android-ui-components-sample` app.
```
Comment (leandroBorgesFerreira) [inline README.md:41]: I'm not sure if we should change to `stream-chat-android-ui-components-sample` at this point.

Comment (zsmb13) [reply README.md:41]: I think this should wait until we officially publish the new UI library

File: README.md [hunk 2]
```diff
@@ -168,10 +173,10 @@ class MainActivity: AppCompatActivity() {
 
 ## Online status
 
-Connection status to Chat is available via `ChatDomain.instance().online` which returns a LiveData object you can attach observers to.
+Connection status to Chat is available via `ChatDomain.instance().isOnline()` which returns a Boolean.
```
Comment (zsmb13) [inline README.md:176]: Why do you want to recommend the one-time check instead of the observable?

Comment (leandroBorgesFerreira) [reply README.md:176]: I thought this has changed. I'll move it back

File: README.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ This repo includes a fully functional example app featuring threads, reactions,
 git clone git@github.com:GetStream/stream-chat-android.git
 ```
 
-Next, download Android Studio and open up the `stream-chat-android` folder. You'll want to run the `stream-chat-android-sample` app.
+Next, download Android Studio and open up the `stream-chat-android-sample` folder. You'll want to run the `stream-chat-android-sample` app.
```
Comment (zsmb13) [inline README.md:41]: The folder name here was correct, that's the name of the folder created when cloning the repo (as it's the name of the repo)

File: README.md [hunk 4]
```diff
@@ -55,10 +55,11 @@ The SDK provides:
 - Offline support and LiveData APIs module
 - Ready to use ViewModels for displaying a list of channels and a conversation 
 - Four reusable chat views:
-    - [Channel List](https://getstream.io/chat/docs/channel_list_view/?language=kotlin)
+    - [Channel List](https://getstream.io/chat/docs/channels_view/?language=kotlin)
     - [Message List](https://getstream.io/chat/docs/message_list_view/?language=kotlin)
     - [Message Input](https://getstream.io/chat/docs/message_input_view/?language=kotlin)
     - [Channel Header](https://getstream.io/chat/docs/channel_header_view/?language=kotlin)
+    - [Message Input View](https://getstream.io/chat/docs/message_input_view/?language=kotlin49)
```
Comment (zsmb13) [inline README.md:62]:
```suggestion
    - [Message Input View](https://getstream.io/chat/docs/message_input_view/?language=kotlin)
```

**Review by zsmb13 — APPROVED** [review]

**Pull request merged**